package.json

```
@@ -4,7 +4,7 @@
   "description": "The SDK for the p2p commons",
   "main": "index.js",
   "scripts": {
-    "test": "node tests/index.js | tap-spec",
+    "test": "node tests/tap.js tests/index.js tests/validate.js | tap-spec",
```

```suggestion
    "test": "tap tests/index.js tests/validate.js | tap-spec",
```

Not sure it's worth it introducing a custom test runner for a small test base

---

Fixed!